Apache-Beam-Jobs.md

```diff
@@ -197,76 +222,58 @@ error_pcoll = (
-
--   **TL;DR**: Inherit from `base_jobs.JobBase` and override the `run()` method.
-
--   The `run()` method must return a `PCollection` of `JobRunResult` instances.
-    -   In English, this means that **the job _must_ report _something_ about
-        what was done during its execution.** This can be the errors it
-        discovered, or the number of successful operations it was able to
-        perform.
-    -   Regardless of your needs, **jobs _must_ report _something_; empty
-        results are forbidden!**
-        -   If you don't think your job has any results worth reporting, then
-            just print a "Success" metric with the number of models it
-            processed.
-    -   `JobRunResult` outputs should answer the following questions:
-        -   Did the job run without any problems? How and why do I know?
-        -   How much work did the job manage to do?
-        -   If the job encountered a problem, what caused it?
-
--   When writing new jobs, prefer splitting boilerplate into new, small, and
-    simple `PTransform` subclasses. Then, after unit testing them, combine them
-    liberally in your job's `run()` method.
-    -   Keep the job class and the `PTransform`s it uses in the same file,
-        unless you plan on reusing them in future jobs. If you _do_ plan on
-        reusing the job, then ask your reviewer for guidance on how to organize
-        it.
-
--   Never modify input values. If you need to make changes to an input value,
-    then [clone it first][2].
-
-Quick Start
-===========
-
-The quick start is split into case studies of increasing complexity. Study the
-one that best suits your needs.
-
-If none of them help you implement your job, you may request a new one by adding
-a comment to [#13190](https://github.com/oppia/oppia/issues/13190) with answers
-to the following questions:
--   Why do I want a new case study?
--   Why are the current case studies insufficient?
--   What answers would the "perfect" case study provide?
-
-Then we'll start write a new Case Study to help you, and future contributors,
-as soon as we can (brianrodri@ will always notify you of how long it'll take).
-
-Case study: `CountAllModelsJob`
--------------------------------
+### `Runner`s
+
+`Runner`s provide the run() method used to visit every node (`PValue`) in the pipeline's DAG by executing the edges (`PTransform`s) to compute their values.  At Oppia, we use `DataflowRunner` to have our `Pipeline`s run on the [Google Cloud Dataflow service](https://cloud.google.com/dataflow).
+
+### High-level Guidelines
+
+**TL;DR**: Inherit from `base_jobs.JobBase` and override the `run()` method.
+
+* The `run()` method must return a `PCollection` of `JobRunResult` instances.
+
+  * In English, this means that **the job _must_ report _something_ about what was done during its execution.** This can be the errors it discovered, or the number of successful operations it was able to perform. **Empty results are forbidden!**
```

Comment:
```suggestion
  * In English, this means that **the job _must_ report _something_ about what was done during its execution.** For example, this can be the errors it discovered or the number of successful operations it was able to perform. **Empty results are forbidden!**
```

Reply:
done